### ISSUE TITLE
Add new mirror or change the Cron got an error

### ISSUE THREAD

**Issue by preussal** [Salamek/gitlab-tools#44]
Hi

when i add a now mirror with a Cron or edit a old mirror and change the Cron then i get this.

```
E0809 10:33:39.827 483829 app.py:1891] Exception on /pull-mirror/edit/40 [POST]
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3/dist-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3/dist-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/lib/python3/dist-packages/flask_login/utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/gitlab_tools/views/pull_mirror/index.py", line 230, in edit_mirror
    if flask.request.method == 'POST' and form.validate():
  File "/usr/lib/python3/dist-packages/gitlab_tools/forms/pull_mirror.py", line 170, in validate
    ExpressionDescriptor(self.periodic_sync.data, throw_exception_on_parse_error=True)
  File "/usr/lib/python3/dist-packages/cron_descriptor/ExpressionDescriptor.py", line 69, in __init__
    raise WrongArgumentException("Unknown {} configuration argument".format(kwarg))
cron_descriptor.Exception.WrongArgumentException: Unknown throw_exception_on_parse_error configuration argument
```

**Comment by preussal** [comment]
the Post command is
![grafik](https://github.com/Salamek/gitlab-tools/assets/26431172/baa402cb-3852-4758-bd54-f134308d2111)

**Comment by Salamek** [comment]
Should be fixed in 1.4.6
